### PR TITLE
IFIX-240-UI : FrameWork - API Exception Handling

### PR DESCRIPTION
**Pull request by krishnaji-eGov**

### CODE REVIEW

**Comment by krishnaji-eGov** [comment]
[Link for ticket](https://digit-discuss.atlassian.net/browse/IFIX-155?atlOrigin=eyJpIjoiZTg3YjhlNWZkMjA5NDUwNWJlYjQ3Yjg3MDBkOWJjNmMiLCJwIjoiaiJ9)

File: frontend/mgramseva/lib/repository/changePassword_details_repo.dart [hunk 1]
```diff
@@ -14,7 +15,7 @@ class ChangePasswordRepository extends BaseService {
     var commonProvider = Provider.of<CommonProvider>(
         navigatorKey.currentContext!,
         listen: false);
-    final requestInfo = RequestInfo('Rainmaker', .01, "", "create", "", "", "",
+    final requestInfo = RequestInfo(APIConstants.API_MODULE_NAME, APIConstants.API_VERSION, APIConstants.API_TS, "create", APIConstants.API_DID, APIConstants.API_KEY, APIConstants.API_MESSAGE_ID,
```
Comment (naveen-egov) [inline frontend/mgramseva/lib/repository/changePassword_details_repo.dart:18]: Create a enum with Methods

File: frontend/mgramseva/lib/services/user.dart [hunk 2]
```diff
@@ -53,7 +54,7 @@ Future<http.Response> login(url, details, context) async {
 }
 
 Future otpforresetpassword(details, context) async {
-  final requestInfo = RequestInfo('Rainmaker', .01, "", "_search", 1, "", "",
+  final requestInfo = RequestInfo(APIConstants.API_MODULE_NAME, APIConstants.API_VERSION, APIConstants.API_TS, "_search", APIConstants.API_DID, APIConstants.API_KEY, APIConstants.API_MESSAGE_ID,
       "2cc113a0-e3c8-4665-9a41-21746e27f2fb");
```
Comment (naveen-egov) [inline frontend/mgramseva/lib/services/user.dart:58]: Value hardcoded

File: frontend/mgramseva/lib/utils/error_logging.dart [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        }
+        navigatorKey.currentState?.pushNamedAndRemoveUntil(Routes.LOGIN, (route) => false);
+        return false;
+      case ExceptionType.INVALIDINPUT: case ExceptionType.OTHER:case ExceptionType.FETCHDATA :case ExceptionType.BADREQUEST :
+      logError(e.message ?? '', stackTrace);
+       return true;
+      case ExceptionType.CONNECTIONISSUE:
+        Notifiers.getToastMessage(
+            context,
+            e.message ?? '${ApplicationLocalizations.of(context).translate(
```
Comment (naveen-egov) [inline frontend/mgramseva/lib/utils/error_logging.dart:33]: Add a  i18translator

**Review by naveen-egov — CHANGES_REQUESTED** [review]
Kindly Make changes as mentioned in comment

**Comment by naveen-egov** [comment]
Please Change the Branch name to Jira reference Ticket ID

**Pull request closed without merging**